operations/workplace/people/offboarding.md

```diff
@@ -1,43 +1,58 @@
+  * Confirm rehire eligibility (if resignation was regrettable, not regrettable).
+  * Confirm whether a backfill is required. If yes, follow instructions to [open a new role](https://handbook.mattermost.com/contributors/join-us/staff-recruiting#how-to-open-a-new-role). 
+
+Thank the departing staff member for their contributions and wish them the best for the future. Every person that joins Mattermost is part of the team and welcome to join our [Community](https://mattermost.com/community/) and stay in touch in whatever capacity feels most comfortable.  
 
-Forward resignation letter/email to MLT member and HR hr@mattermost.com. The resignation letter/email will be stored in Bamboo.
+HR will reach out to the departing staff member to communicate last day instructions and details and to schedule an [exit interview](https://docs.google.com/document/d/1MX6LXa2uVEM3_xqH-pqKyJH4PWis9oJ0a5sAHd58W1U/edit?ts=5e99ecc0).
 
-A resignation letter/email is required from departing staff. In most cases and depending upon the statutory requirements in the staff member’s country/region, a minimum of two weeks' notice is required for voluntary departures.
+## Instructions for Departing Team Members
 
-HR will reach out to the departing staff member to communicate last day instructions and details and to schedule an [exit interview](https://docs.google.com/document/d/1MX6LXa2uVEM3_xqH-pqKyJH4PWis9oJ0a5sAHd58W1U/edit?ts=5e99ecc0).
+1. Forward resignation letter/email to MLT member and HR hr@mattermost.com. The resignation letter/email will be stored in Bamboo.
+Note: a resignation letter/email is required from departing staff. In most cases and depending upon the statutory requirements in the staff member’s country/region, a minimum of two weeks' notice is required for voluntary departures.
+2. Work with your Manager to develop a [transition plan]([Communication Plan](https://handbook.mattermost.com/operations/workplace/people/offboarding#plan-for-transition) and [Communication Timeline](https://handbook.mattermost.com/operations/workplace/people/offboarding#communicating-departures). Departing Team Members should feel encouraged to say goodbye to peers,  We encourage team members to [make meaninful connections](https://handbook.mattermost.com/contributors/join-us/recruiting#4-make-meaningful-connections) and encourage departing team members to say goodbye to peers, or optionally post in the [Stand-up Channel](https://community.mattermost.com/private-core/channels/stand-up). We just ask to align with your Manager on timeline of messaging so that direct teammates or working groups can be informed with an adequate plan on  
```

Comment:
```suggestion
2. Work with your Manager to develop a [transition plan]([Communication Plan](https://handbook.mattermost.com/operations/workplace/people/offboarding#plan-for-transition) and [Communication Timeline](https://handbook.mattermost.com/operations/workplace/people/offboarding#communicating-departures). We encourage team members to [make meaningful connections](https://handbook.mattermost.com/contributors/join-us/recruiting#4-make-meaningful-connections) and encourage departing team members to say goodbye to peers, or optionally post in the [Stand-up Channel](https://community.mattermost.com/private-core/channels/stand-up). We just ask to align with your Manager on timeline of messaging so that direct teammates or working groups can be informed with an adequate plan ahead of departure.
```